beacon/beacon_test.go

```diff
@@ -730,41 +730,6 @@ func TestBeacon_buildProposal(t *testing.T) {
 	}
 }
 
-func TestBeacon_signMessage(t *testing.T) {
```

Comment:
why rm this test?

Reply:
gonna add separate tests for different message types. scale codec doesn't work with `interface{}`

Reply:
i am not sure this test is meaningful anyway....

Reply:
true, I'll think if I can make it :)